src/librustdoc/clean/utils.rs

```diff
@@ -73,7 +73,7 @@ pub(crate) fn krate(cx: &mut DocContext<'_>) -> Crate {
         }));
     }
 
-    Crate { module, primitives, external_traits: cx.external_traits.clone() }
+    Crate { module, external_traits: cx.external_traits.clone() }
```

Comment:
hmm, I'm not useful this change is since you still call `crate.primitives()`, you just don't store it afterwards.

Reply:
I tried to cache `ExternalCrate::primitives` as a OnceCell like `primitive_locations`, but ran into issues since `primitives` is called for multiple different crates, so you can't use OnceCell and call it a day.